electricitymap/contrib/capacity_parsers/README.md

```diff
@@ -0,0 +1,180 @@
+# Capacity update process <a name="#top"></a>
+
+## Table of contents
+
+- [Context](#context)
+- [Capacity source](#capacity_sources)
+- [Capacity update process](#capacity_update_process)
+
+  - [When should capacity data be updated?](#when_to_update)
+  - [Format of the capacity configuration](#format)
+  - [Opening a PR](#PR)
+  - [The zone capacity can be updated automatically](#automatic_update)
+  - [The zone capacity is updated manually](#manual_update)
+
+- [Technical requirements for adding a new data source](#technical_requirements)
+- [Building a new capacity parser](#building_a_parser)
+
+## Context <a name="#context"></a>
+
+In an effort to increase the quality of the data published on our app or API, we have started a whole initiative to enable us to track outliers in the source data.
+
+Over the years, we have noticed that real-time data published by the different data sources can be completely outside the distribution of the historical time-series. These data points are outliers and need to be detected before any data processing.
+
+One way to perform outlier detection is to check that each incoming data point is not higher than the installed capacity for a given mode. The power output cannot be above the energy input as the efficiency of power plants is always below 100%. In other words, for a given zone and a given mode, the power production for that mode at any given time cannot exceed the installed capacity for that mode.
+
+> **Example**
+> In 2023, the wind capacity in **DK-DK1** was 5233 MW. The average wind production in the first 3 quarters of 2023 was 1455 MW.
+
+The goal here is to validate each incoming production parser event by comparing each mode production against the installed capacity available in the zone configuration. If a mode production is higher than the installed capacity, the data point will be flagged as an outlier and will be corrected by our data pipelines.
+
+To achieve this goal, we need robust and consistent capacity data. We also need to be able to capture the evolution of capacity data over time. As renewable capacity increases in most zones, this means that the power production will also increase.
```

Comment:
Maybe we should just reference the wiki for this section? Think it's less important to also have it here :)

Reply:
will do :)